corehq/motech/repeaters/dbaccessors.py

```diff
@@ -185,6 +185,17 @@ def _iter_repeat_records_by_repeater(domain, repeater_id, chunk_size,
             yield doc['id']
 
 
+def iter_repeaters():
```

Comment:
Developers have been frustrated in the past when they discover that `get_foo()` returns a generator, instead of a list or tuple that can be iterated more than once. So I use `iter_foo()` to make callers aware that they aren't getting a list.

Maybe a type hint would check both boxes?
```python
def get_all_repeaters() -> Generator:
    ...
```